src/abstractdataframe/abstractdataframe.jl

```diff
@@ -410,6 +410,29 @@ function Base.isequal(df1::AbstractDataFrame, df2::AbstractDataFrame)
     return true
 end
 
+function Base.isapprox(df1::AbstractDataFrame, df2::AbstractDataFrame;
+                    atol::Real=0, rtol::Real=atol>0 ? 0 : √eps(),
+                    nans::Bool=false, norm::Function=abs)
```

Comment:
```suggestion
                    nans::Bool=false, norm::Function=norm)
```
As discussed I suggest to just use the default from `LinearAlgebra`.
Please just update Project.toml and import `norm` from `LinearAlgebra`.